plugins/catalog-backend/src/service/standaloneServer.ts

```diff
@@ -29,21 +32,19 @@ export async function startStandaloneServer(
   options: ServerOptions,
 ): Promise<Server> {
   const logger = options.logger.child({ service: 'catalog-backend' });
+  
+  const database = knex({
+    client: 'sqlite3',
+    connection: ':memory:',
+    useNullAsDefault: true,
+  });
+  database.client.pool.on('createSuccess', (_eventId: any, resource: any) => {
+    resource.run('PRAGMA foreign_keys = ON', () => {});
+  });
```

Comment:
i actually thought that this could be in DatabaseManager, so that `createInMemoryDatabase(logger)` is a oneliner to get an entire Database created, including the setup of the knex sqlite stuff